src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -346,8 +354,10 @@ const WrappedArtworkApp: React.FC<React.PropsWithChildren<Props>> = props => {
   // Check to see if referrer comes from link interception.
   // @see interceptLinks.ts
   const referrer = state && state.previousHref
+  const isLotClosed = lotIsClosed(sale, saleArtwork)
 
-  const websocketEnabled = !!sale?.extendedBiddingIntervalMinutes
+  const websocketEnabled =
+    !!sale?.extendedBiddingIntervalMinutes && isLotClosed === false
```

Comment:
Haven't totally grokked this, but does this imply that a sale without extended bidding will always skip polling?

Reply:
This should probably be ||